app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
+  default from: "from@example.com"
```

Comment:
This is probably something we want to get from someone from the RubyGems team and maybe not hardcode it, but get if from a config or ENV var.